samples/net/cloud/mqtt_azure/src/main.c

```diff
@@ -220,12 +237,53 @@ static void mqtt_event_handler(struct mqtt_client *const client,
 		LOG_DBG("PUBACK packet id: %u\n", evt->param.puback.message_id);
 		break;
 
+	case MQTT_EVT_PUBLISH:
+		len = evt->param.publish.message.payload.len;
+
+		LOG_INF("MQTT publish received %d, %d bytes", evt->result, len);
+		LOG_INF(" id: %d, qos: %d", evt->param.publish.message_id,
+			evt->param.publish.message.topic.qos);
+
+		while (len) {
+			bytes_read = mqtt_read_publish_payload(&client_ctx,
+						data, len >= 32 ? 32 : len);
```

Comment:
Instead of hardcoded value of 32, do `sizeof(data) - 1` instead

Reply:
Done